testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -456,7 +458,12 @@ impl ClusterSwarmKube {
         let (p, s): (Pod, Service) = match &instance_config.application_config {
             Validator(validator_config) => (
                 self.validator_spec(
-                    instance_config.validator_group,
+                    instance_config.validator_group.index as u32,
+                    match instance_config.validator_group.twin_index {
+                        None => format!("val-{}", instance_config.validator_group.index),
+                        _ => format!("val-{}-twin", instance_config.validator_group.index),
```

Comment:
Also, since twin index is a number, it should be part of this too? Can be smth like `val-{}-twin-{}`